runtime/runtime/src/prefetch.rs

```diff
@@ -121,12 +160,44 @@ impl TriePrefetcher {
     fn prefetch_trie_key(&self, trie_key: TrieKey) -> Result<(), ()> {
         let queue_full = self.prefetch_api.prefetch_trie_key(self.trie_root, trie_key).is_err();
         if queue_full {
+            self.prefetch_queue_full.inc();
             debug!(target: "prefetcher", "I/O scheduler input queue full, dropping prefetch request");
             Err(())
         } else {
+            self.prefetch_enqueued.inc();
             Ok(())
         }
     }
+
+    /// Prefetcher specifically tuned for SWEAT record batch
+    fn prefetch_sweat_record_batch(&self, account_id: AccountId, arg: &[u8]) -> Result<(), ()> {
```

Comment:
Please comment that this function is temporary and add a link to the flat-storage github issue.

Reply:
done